poetwp.php

```diff
@@ -30,53 +29,32 @@
 	die;
 }
 
-/**
- * Currently plugin version.
- * Start at version 1.0.0 and use SemVer - https://semver.org
- * Rename this for your plugin and update it as you release new versions.
- */
-define( 'POETWP_VERSION', '1.0.0' );
+// Include the file that defines the custom post type functions.
+require_once plugin_dir_path( __FILE__ ) . 'includes/post-type-functions.php';
 
-/**
- * The code that runs during plugin activation.
- * This action is documented in includes/class-poetwp-activator.php
- */
-function activate_poetwp() {
-	require_once plugin_dir_path( __FILE__ ) . 'includes/class-poetwp-activator.php';
-	Poetwp_Activator::activate();
-}
+// Include the file that creates demo poem.
+require_once plugin_dir_path( __FILE__ ) . 'includes/demo-poem.php';
 
-/**
- * The code that runs during plugin deactivation.
- * This action is documented in includes/class-poetwp-deactivator.php
- */
-function deactivate_poetwp() {
-	require_once plugin_dir_path( __FILE__ ) . 'includes/class-poetwp-deactivator.php';
-	Poetwp_Deactivator::deactivate();
-}
+// Include the file that defines the admin functions.
+require_once plugin_dir_path( __FILE__ ) . 'admin/admin-functions.php';
 
-register_activation_hook( __FILE__, 'activate_poetwp' );
-register_deactivation_hook( __FILE__, 'deactivate_poetwp' );
+// Include the file that defines the public functions.
+require_once plugin_dir_path( __FILE__ ) . 'public/public-functions.php';
 
-/**
- * The core plugin class that is used to define internationalization,
- * admin-specific hooks, and public-facing site hooks.
- */
-require plugin_dir_path( __FILE__ ) . 'includes/class-poetwp.php';
+// Register activation hook.
+register_activation_hook( __FILE__, 'poetwp_activate' );
```

Comment:
No change needed here, but just for fun, plugins activated programmatically on WPVIP don't get their activation callback called...something to think about 😉